### ISSUE TITLE
Publish arm64-Darwin releases

### ISSUE THREAD

**Issue by sgtsquiggs** [casey/just#1132]
When using the install script on an Apple M1 machine you run across:

install: error: Could not determine target from output of `uname -m`-`uname -s`, please use `--target`: arm64-Darwin

Can this be published? I can currently install via brew or manually but the script was useful for local CICD.

**Comment by casey** [comment]
Thanks for opening this issue! This is blocked on Github Actions releasing arm64 MacOS runners. I think this is the issue for that: https://github.com/actions/runner/issues/805

Once arm64 runners are available, this should be very easy.

**Comment by messense** [comment]
FYI, you don't need arm64 macOS runners to build for M1 Mac, Rust supports cross compile to aarch64-apple-darwin on x86_64 macOS:

```
❯ uname -a
Darwin messense.local 21.3.0 Darwin Kernel Version 21.3.0: Wed Jan  5 21:37:58 PST 2022; root:xnu-8019.80.24~20/RELEASE_X86_64 x86_64

❯ cargo build --target aarch64-apple-darwin
    Updating crates.io index
  Downloaded nix v0.23.1
  Downloaded regex v1.5.5
  Downloaded typed-arena v2.0.1
  Downloaded strum_macros v0.24.0
  Downloaded strum v0.24.0
  Downloaded snafu-derive v0.7.0
  Downloaded target v2.0.0
  Downloaded edit-distance v2.1.0
  Downloaded lexiclean v0.0.1
  Downloaded ctrlc v3.2.1
  Downloaded dotenv v0.15.0
  Downloaded snafu v0.7.0
  Downloaded similar v2.1.0
  Downloaded rustversion v1.0.6
  Downloaded 14 crates (705.2 KB) in 3.43s
   Compiling proc-macro2 v1.0.36
   Compiling unicode-xid v0.2.2
   Compiling syn v1.0.86
   Compiling libc v0.2.119
   Compiling memchr v2.4.1
   Compiling cfg-if v1.0.0
   Compiling autocfg v1.1.0
   Compiling rustversion v1.0.6
   Compiling serde_derive v1.0.136
   Compiling log v0.4.14
   Compiling serde v1.0.136
   Compiling doc-comment v0.3.3
   Compiling unicode-segmentation v1.9.0
   Compiling bitflags v1.3.2
   Compiling regex-syntax v0.6.25
   Compiling camino v1.0.7
   Compiling serde_json v1.0.79
   Compiling lazy_static v1.4.0
   Compiling unicode-width v0.1.9
   Compiling heck v0.4.0
   Compiling regex-automata v0.1.10
   Compiling ansi_term v0.12.1
   Compiling itoa v1.0.1
   Compiling strsim v0.8.0
   Compiling humantime v2.1.0
   Compiling ryu v1.0.9
   Compiling fastrand v1.7.0
   Compiling vec_map v0.8.2
   Compiling termcolor v1.1.3
   Compiling remove_dir_all v0.5.3
   Compiling target v2.0.0
   Compiling typed-arena v2.0.1
   Compiling edit-distance v2.1.0
   Compiling dotenv v0.15.0
   Compiling lexiclean v0.0.1
   Compiling memoffset v0.6.5
   Compiling heck v0.3.3
   Compiling aho-corasick v0.7.18
   Compiling bstr v0.2.17
   Compiling quote v1.0.15
   Compiling similar v2.1.0
   Compiling regex v1.5.5
   Compiling atty v0.2.14
   Compiling term_size v0.3.2
   Compiling nix v0.23.1
   Compiling tempfile v3.3.0
   Compiling textwrap v0.11.0
   Compiling clap v2.34.0
   Compiling env_logger v0.9.0
   Compiling ctrlc v3.2.1
   Compiling strum_macros v0.24.0
   Compiling snafu-derive v0.7.0
   Compiling derivative v2.2.0
   Compiling strum v0.24.0
   Compiling snafu v0.7.0
   Compiling just v1.1.0 (/Users/messense/Projects/just)
    Finished dev [unoptimized + debuginfo] target(s) in 1m 08s

❯ file target/aarch64-apple-darwin/debug/just
target/aarch64-apple-darwin/debug/just: Mach-O 64-bit executable arm64
```

**Comment by casey** [comment]
Nice, I didn't know that! I just released 1.1.1, which should include MacOS ARM binaries, once the build finishes: https://github.com/casey/just/releases/tag/1.1.1

**Comment by casey** [comment]
Just confirmed that the install script works on my M1 Mac!
